WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -903,6 +994,40 @@ + (BOOL)isNewEditorEnabled
 
 #pragma mark - Instance Methods
 
+- (void)keyboardWillHide:(NSNotification*)aNotification
+{
+    // If the format bar tooltip is still hanging around, let's git rid of it
+    if (self.formatBarToolTip) {
+        [self.formatBarToolTip cancelCurrentTooltip];
+        self.formatBarToolTip = nil;
+    }
+}
+
+- (void)keyboardWasShown:(NSNotification*)aNotification
+{
+    NSDictionary* info = [aNotification userInfo];
+    CGRect rawKeyboardRect = [[info objectForKey:UIKeyboardFrameEndUserInfoKey] CGRectValue];
+    CGRect kRect = [self.view.window convertRect:rawKeyboardRect toView:self.view.window.rootViewController.view];
```

Comment:
`showFormatBarOnboarding` displays the tooltip in this view controller's view, so I'm not sure why we'd want to trust a rect converted from a different view controllers view, even if the dimensions are the same.